x-pack/legacy/plugins/integrations_manager/public/screens/detail/screenshots.tsx

```diff
@@ -55,11 +55,15 @@ export function Screenshots(props: ScreenshotProps) {
           </NestedEuiFlexItem>
         )}
         <NestedEuiFlexItem>
```

Comment:
@neptunian I see that you're doing a lot of overrides on top of the EuiFlexGroup/Items. I thin at this point using these EUI components is unnecessary since you are overriding the flex properties. It **may** also be why you're having issues in IE. I would suggest using plain `div`'s and just adding the flex properties you need.

Though when I look at the FlexGroup properties, you're using it as a column layout which is also the standard block layout (top to bottom). Maybe consider just relying on the normal div/box layout?

Reply:
I did try removing many of the flex items but the problem persisted all the way outside of our plugin to a flex div with a class called `.app-wrapper-panel`.